### ISSUE TITLE
Accessibility - Missing Heading Text on "Criminal" Tab of the Enforcement Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#63]
When I was testing the application, I noticed on the "Criminal" tab of the Enforcement screen there is missing heading text. My guess is input file is missing the information or not pulling it to the screen correctly.

**Comment by aquilent-dgemoets** [comment]
Do you mean the white horizontal bar with no text?  That's just a divider to separate the subsequent lines from the previous section-- it is not missing a header or any header text.

**Comment by jfc3-dol** [comment]
It's a gray horizontal line that other ones have text in them. I wanted to make sure we were not missing anything is all.

**Comment by aquilent-dgemoets** [comment]
Yep-- I think that's working as intended.  Thanks for checking.  I think we can close the issue.

**Comment by jfc3-dol** [comment]
Closing issue then.
